### ISSUE TITLE
Status of Swift Mailer support in v2.0.0?

### ISSUE THREAD

**Issue by ThomasLandauer** [Codeception/module-symfony#125]
@TavoNiievez Am I right that the `mailer` configuration you/we introduced in https://github.com/Codeception/module-symfony/pull/9/files was removed in v2.0.0, and the module just doesn't work with Swift Mailer anymore?

At https://github.com/Codeception/module-symfony/releases/tag/2.0.0 you only mentioned that Symfony 4.4 is required, but this doesn't automatically mean that Swift Mailer was dropped in favor of Symfony Mailer, cause Swift Mailer is still supported in Symfony 4.4: https://symfony.com/doc/4.4/email.html

This is the "Fail" message people get when running their tests:

> Emails can't be tested without Symfony Mailer service.

And I'd say it should be reworded into something like:

> `codeception/module-symfony` requires Symfony Mailer to test emails. So you need to either upgrade your app from Swift Mailer to Symfony Mailer https://symfony.com/doc/current/mailer.html or downgrade `codeception/module-symfony` to 1.6.0

... cause some people might not have heard of Symfony Mailer at all yet ;-)

But it would be better to inform people *upfront* that it's not going to work. How could we do that? Maybe a yellow box at https://codeception.com/docs/modules/Symfony somewhere near the top? I can come up with some PR, but want to hear your opinion first :-)

BTW: On that page, what is that blue "table" in the top right: Changelog | Source | 3.1 | 2.5 | 1.8 ? The version numbers look like Codeception's, so they're quite misleading here. Do you know how/where this can be changed?

**Comment by TavoNiievez** [comment]
@ThomasLandauer As always, thanks for opening.

You're right.

I updated [the release notes](https://github.com/Codeception/module-symfony/releases/tag/2.0.0) to include that explanation. Is it okay as it is?

> And I'd say it should be reworded (the "Fail" message)

Yes, you're right, it seems like something important. The idea is that if you use Swift Mailer and you don't want (or can't) update to Symfony Mailer just stay on the 1.x branch of this module, please send a PR... the message you mention is fine for me.

> Do you know how/where this can be changed? (the blue "table" in the top right)

It seems to come from [this file](https://github.com/Codeception/codeception.github.com/blob/1568f32c874a3eb6fc48b5ea2448c4dfca73cf84/docs/modules/Symfony.md). That part shouldn't be overwritten when generating the documentation automatically, as far as I know.

**Comment by TavoNiievez** [comment]
> what is that blue "table" in the top right: Changelog | Source | 3.1 | 2.5 | 1.8 ?

I think those numbers were referring to the version of Codeception, when this module was integrated into the main project, but the new version (the one in this repo) seems that it was never added there.

**Comment by Naktibalda** [comment]
> BTW: On that page, what is that blue "table" in the top right: Changelog | Source | 3.1 | 2.5 | 1.8 ? The version numbers look like Codeception's, so they're quite misleading here. Do you know how/where this can be changed?

Yes, they are Codeception version numbers, and the buttons are generated by documentation page generator, so they are the same for all modules.

Codeception had links to older versions of documentation probably since 2.0 release (If I remember correctly, they were to 1.6 and 1.8 versions originaly), so I replaced them with links to last minor versions when I extracted modules to separate repositories.

I don't know if they ever were useful to anyone, but I think that they can be removed now.
The code which generates buttons is at https://github.com/Codeception/codeception.github.com/blob/master/RoboFile.php#L162-L168
